IdentityCore/src/cache/crypto/MSIDAssymetricKeyGeneratorFactory.m

```diff
@@ -40,15 +42,17 @@ @implementation MSIDAssymetricKeyGeneratorFactory
 
 + (id<MSIDAssymetricKeyGenerating>)iOSDefaultKeyGeneratorWithError:(NSError **)error
 {
-    return [[MSIDAssymetricKeyKeychainGenerator alloc] initWithGroup:nil error:error];
+    NSString *keychainGroup = [[MSIDKeychainTokenCache defaultKeychainCache] keychainGroup];
+    return [[MSIDAssymetricKeyKeychainGenerator alloc] initWithGroup:keychainGroup error:error];
 }
 
 #if !TARGET_OS_IPHONE
 + (id<MSIDAssymetricKeyGenerating>)macDefaultKeyGeneratorWithError:(NSError **)error
 {
     if (@available(macOS 10.15, *))
     {
-        return [[MSIDAssymetricKeyKeychainGenerator alloc] initWithGroup:nil error:error];
+        NSString *keychainGroup = [[MSIDMacKeychainTokenCache defaultKeychainCache] keychainGroup];
```

Comment:
we need to also pass access ref for login keychain group below

Reply:
Addressed.